browser/create-template/templates/nextjs-site/src/atomic/getAllBlogposts.ts

```diff
@@ -0,0 +1,18 @@
+import { env } from '@/env';
+import { website } from '@/ontologies/website';
+import { CollectionBuilder, core, Store } from '@tomic/lib';
+
+export async function getAllBlogposts(): Promise<string[]> {
+  const store = new Store({
```

Comment:
This logic is probably reusable. I also see it in `app/store.ts`. Should probably import that.